tools/analyzer_plugin/lib/src/component_usage.dart

```diff
@@ -40,6 +42,29 @@ class FluentComponentUsage {
     return null;
   }
 
+  /// Returns the name of the `UiComponent` class instance for the usage, complete with any
+  /// [namespaceName] if one exists.
+  String get componentClassTypeName {
```

Comment:
These implementations aren't valid for DOM components, and should either be updated to handle them, or documented as such